recipes/floret/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "floret" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/explosion/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 9f4c2c0abbee7945b1787c0b4fc46a53b794179f1f10a63feee249b971a2ccae
+  patches:
+    - 0001_do_not_hardcode_compiler.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - make
+    - numpy >=1.17
```

Don't add a `>=` otherwise this may not take the version from the global pinning.
```suggestion
    - numpy
```